### ISSUE TITLE
aclocal: error: 'configure.ac' is required - On openSUSE Tumbleweed

### ISSUE THREAD

**Issue by SeriousHoax** [sarim/ibus-avro#217]
Hi! I'm trying to install it in openSUSE Tumbleweed (KDE).
Installed ibus, git, etc. and cloned the repo but when I enter, `aclocal && autoconf && automake --add-missing`
I'm getting `aclocal: error: 'configure.ac' is required`
![image](https://github.com/user-attachments/assets/91253e45-3c3a-4262-b40d-016a17a08097)
Just as a guess I downloaded "configure.ac" from this repo and put it in the folder and tried the command again and this time I got this,
![image](https://github.com/user-attachments/assets/e307e37b-3c7e-49f1-b087-29b1073bd39c)
What can I do to finish building and installing it in openSUSE Tumbleweed? Can you please suggest something?

**Comment by sarim** [comment]
you failed to clone the repo so files are missing. Make sure you followed the readme and have downloaded the files properly via git, which is called cloning in git.

**Comment by SeriousHoax** [comment]
> you failed to clone the repo so files are missing. Make sure you followed the readme and have downloaded the files properly via git, which is called cloning in git.

@sarim I did it actually. It's not in the screenshot above because that was done about couple of hours before I created this issue. The screenshots that I shared here are the one I took for reporting here only.

**Comment by sarim** [comment]
> Just as a guess I downloaded "configure.ac" from this repo and put it in the folder and tried the command again and this time I got this

If was properly cloned then files like these wouldn't be missing.

**Comment by archisman-panigrahi** [comment]
I just verified running the commands in ubuntu 24.04, and they do work

**Comment by SeriousHoax** [comment]
> > Just as a guess I downloaded "configure.ac" from this repo and put it in the folder and tried the command again and this time I got this
> 
> If was properly cloned then files like these wouldn't be missing.

Okay, let me try again and I'll let you know here. Keep the issue open for now.

**Comment by SeriousHoax** [comment]
@sarim You were right. The issue was indeed in my git. Only git-core was installed on my system instead of the full package. Now everything worked perfectly following the instruction. 
FYI, in case it helps someone else in the future, in openSUSE Tumbleweed I also needed to install Bangla fonts which I didn't need for Ubuntu and Arch based Manjaro and EndeavourOS could be because I didn't install Office tools (Libre Office) in Tumbleweed but I'm not entirely sure. 
Installed from here: `https://github.com/fahadahammed/linux-bangla-fonts`
I'm closing the issue.
